src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -796,6 +796,9 @@ def _find_split():
 def set_properties(instance, expression, force_string):
     key, value = _split_key_value_pair(expression)
 
+    if key.strip() == '':
```

Comment:
Can key be None object ?

Reply:
> Can key be None object ?

If key is None type, key.strip() will raise AttributeError.

Reply:
I have one question about the key value. Although I cannot see the debug information, from your conversation in issue #10464, the error is cause by the space in key value, right? If yes, can we do some check in command/argument validation?

Reply:
> 
> 
> Can key be None object ?

Yes. We should pay attention to corner case.

Reply:
> 
> 
> I have one question about the key value. Although I cannot see the debug information, from your conversation in issue #10464, the error is cause by the space in key value, right? If yes, can we do some check in command/argument validation?

Yes, we can. But it's a generic update command and set parameter. The logic has been already defined in arm.py in core module.